src/test/scala/com/evolutiongaming/scache/CacheSpec.scala

```diff
@@ -456,9 +455,9 @@ class CacheSpec extends AsyncFunSuite with Matchers {
           fiber1    <- cache.getOrUpdateReleasable(0) { Async[IO].never }.startEnsure
```

Comment:
This test fails now. To me at the first glance it looks like the following scenario:
- `fiber0` waits until there's something in `deferred0`
- we cancel `fiber0`, it stops waiting
- we complete `deferred0`, but nothing gets put into the cache as `fiber0` is already canceled
- we try to join on `fiber1` which it turn waits for `Async[IO].never` which never completes. `fiber1` can't take the value from the cache as `fiber0` was prevented from putting it there in the first place